great_expectations/render/renderer/content_block/content_block.py

```diff
@@ -294,6 +312,28 @@ def _render_other(  # noqa: PLR0912, PLR0913
                     result.append(expectation_notes)
         return result
 
+    @classmethod
+    def _render_expectation_description(
+        cls,
+        configuration: ExpectationConfiguration,
+        runtime_configuration: dict,
+        **kwargs,
+    ) -> list[RenderedStringTemplateContent]:
+        expectation = configuration.to_domain_obj()
+        description = expectation.description
```

Comment:
Will passing `None` to `RenderedStringTemplateContent` be ok or should this be `description = expectation.description or ""`?

I know we are only using this in 1 code path right now which has a non-None description.

Reply:
We should be guaranteed a non-null value - I've added a small check